composer.json

```diff
@@ -54,7 +54,7 @@
         "symfony/web-profiler-bundle": "^4.2",
         "symfony/web-server-bundle": "^4.2",
         "symplify/coding-standard": "^5.5",
-        "symplify/easy-coding-standard": "^5.5",
+        "symplify/easy-coding-standard": "^5.4",
```

Comment:
Why this?

Reply:
It is here just to pass travis checks, I'll remove it ;-)

Reply:
Ah, I see it now. Keep it, I'll fix it after merge :)